riot.py

```diff
@@ -109,3 +110,87 @@ def previous_match(name: str):
         participants.append(participants_row)
     last_match_info = pd.DataFrame(participants)
     return last_match_info
+
+
+def create_summoner_list(players_list: list, server_id: str):
+    """Gets the list of summoners and returns the information abou the summoners
+    Parameters:
+    players_list (list): list of summoner names
+    server_id (int): discord server id
+
+    Returns:
+    members_to_add (dict): summoner's latest match information
+
+    """
+
+    # dictionary for data input
+    members_to_add = {
+        server_id: [],
+    }
+
+    # accessing each player
+    for summoner in players_list:
+
+        user = watcher.summoner.by_name(MY_REGION, summoner)
+
+        user_name = user["name"]
+
+        ranked_stat = watcher.league.by_summoner(MY_REGION, user["id"])
+
+        # check if there is any rank data on user as unranked players don't have any data on ranked
+        if len(ranked_stat) > 0:
+
+            solo_rank_stat = pydash.find(ranked_stat, {"queueType": "RANKED_SOLO_5x5"})
+
+            tier_division = solo_rank_stat["tier"]
+            tier_rank = solo_rank_stat["rank"]
+
+            tier_rank_map = {"I": "1", "II": "2", "III": "3", "IV": "4"}
```

Comment:
this should be ported out. We will be using this in different functions too.

Reply:
Resolved by defining in separate file: `utils/constants.py`